schema.sql

```diff
@@ -0,0 +1,287 @@
+CREATE TABLE
+  account (
+    id VARCHAR PRIMARY KEY,
+    total_donations_usd DECIMAL(10, 2),
+    total_matching_pool_allocations_usd DECIMAL(10, 2),
+    donors_count INT
+  );
+
+CREATE TABLE
+  list (
+    id VARCHAR PRIMARY KEY,
+    owner_id VARCHAR NOT NULL,
+    name VARCHAR NOT NULL,
+    description TEXT,
+    default_registration_status ENUM(
+      'Pending',
+      'Approved',
+      'Rejected',
+      'Graylisted',
+      'Blacklisted'
+    ) NOT NULL,
+    FOREIGN KEY (owner_id) REFERENCES account (id)
+  );
+
+CREATE TABLE
+  list_admin (
+    list_id INT NOT NULL,
+    admin_id VARCHAR NOT NULL,
+    PRIMARY KEY (list_id, admin_id),
+    FOREIGN KEY (list_id) REFERENCES list (id),
+    FOREIGN KEY (admin_id) REFERENCES account (id)
+  );
+
+CREATE TABLE
+  list_registration (
+    id SERIAL PRIMARY KEY,
```

Comment:
change this to INT as we will want to manually set it as per registration ID on list contract (which will be an int, but may not increment identically and so we want to make sure there are no discrepancies)

Reply:
registration ID is not returned in the register action, the id in the action result is the project id provided by the user, or the signerID.

Reply:
Isn't the Registration returned in the new Lists contract? I think since the Lists contract will be out within a day or so and all existing projects on registry.potlock.near will be migrated there via the register method, we don't have to account for the current registry contract.